packages/wrangler/src/__tests__/helpers/mock-oauth-flow.ts

```diff
@@ -105,11 +105,53 @@ export const mockOAuthFlow = () => {
     return outcome;
   };
 
+  const mockExchangeRefreshTokenForAccessToken = ({
+    respondWith,
+  }: {
+    respondWith: "refreshSuccess" | "refreshError" | "badResponse";
+  }) => {
+    fetchMock.mockOnceIf(
+      (req) =>
+        (req as Request).url === "https://dash.cloudflare.com/oauth2/token" &&
+        (req as Request).headers.get("Content-Type") ===
+          "application/x-www-form-urlencoded",
```

Comment:
I think we worked out that all the calls to `oauth/token` will be with the `x-www-form-urlencoded` content type, so there is no need to use a predicate function here. Instead we can just use the URL matcher.